tests/src/lib.rs

```diff
@@ -54,6 +54,18 @@ pub struct ConsensusConfig {
     pub timeout: Duration,
 }
 
+impl ConsensusConfig {
+    pub fn happy(n_participants: usize) -> Self {
+        Self {
+            n_participants,
+            threshold: Fraction::one(),
+            // Since it takes 1+ secs to start each nomos-node,
+            // set the timeout conservatively.
+            timeout: Duration::from_millis(n_participants as u64 * 2500),
```

Comment:
this is only for the chain topology, for start we could start all nodes concurrently

Reply:
Yes, I think so. But, I thought it's not bad to set a large timeout regardless of the topology because we anyway don't expect timeout. If timeout happens, the test will be failed by the test timeout (20s).

Reply:
Hm. Nevertheless, the way you told sounds better. Let me try it.

Reply:
I agree with raising the timeout for happy path since as you said we check that there are no timeouts anyway, I was just pointing out that this `happy` configuration only makes sense if you use the `Chain` topology but from the api it seems it's a general config. Not too much of an issue anyway since we usually have test wide timeouts

Reply:
In short, what I'm suggesting is to change this to be a method on `SpawnConfig`, something like `SpawnConfig::happy_chain(n_participants: u64, mixnet_config: ...)`

Reply:
I get it. Yes. The API (`ConsensusConfig::happy()`) can mislead. I'll refactor it now.

Reply:
Refactored: e59f8da. Thank you for reviewing this carefully!